tests/test_aea.py

```diff
@@ -170,16 +164,17 @@ def test_react():
         )
         builder.set_default_connection(PublicId.from_str("fetchai/local:0.1.0"))
         builder.add_skill(Path(CUR_PATH, "data", "dummy_skill"))
-        aea = builder.build(connection_ids=[PublicId.from_str("fetchai/local:0.1.0")])
+        agent = builder.build(connection_ids=[PublicId.from_str("fetchai/local:0.1.0")])
```

Comment:
As a rule aiding simpler reading, since we have both `AEA` and `Agent` it is best to refer to the class you're using, here `aea`